content/docs/developing-operators/plans.md

```diff
@@ -140,3 +140,38 @@ parameters:
 - name: APPLICATION_MEMORY
   trigger: canary
 ```
+
+## Executing Plans
```

Comment:
In know this feature has been running as _Manual Plan Execution_ so far. I still want to suggest rethinking how the docs present this. I'd argue that the execution is not manual – plans are executed by KUDO. The thing that is manually do is the trigger: _Triggering Plans_ might be a better title.

This also maps the CLI/yaml where you can either specify a triggering parameter, or _trigger_ manually.

WDYT?

Reply:
Agree!